documentation/modules/max-concurrent-vms.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: CONCEPT
+[id="max-concurrent-vms_{context}"]
+= A detailed look at controller_max_vm_inflight
+
+For {rhv-full}, {osp}, vmWare warm migrations _or_ remote cold migrations, the setting `controller_max_vm_inflight` (shown in the UI as *Max concurrent virtual machine migrations*) specifies the maximum number of _disks_ that can be transferred simultaneously. In these migrations, the disks are migrated in parallel. This means that if the combined number of disks that you want to migrate is greater than the value of the setting, additional disks wait until the queue is free without regard for whether a VM has completely finished being migrated.
```

Comment:
```suggestion
For {rhv-full}, {osp}, VMware warm migrations _or_ remote cold migrations, the setting `controller_max_vm_inflight` (shown in the UI as *Max concurrent virtual machine migrations*) specifies the maximum number of _disks_ that can be transferred simultaneously. In these migrations, the disks are migrated in parallel. This means that if the combined number of disks that you want to migrate is greater than the value of the setting, additional disks wait until the queue is free without regard for whether a VM has completely finished being migrated.
```

Remember the correct format for [VMware](https://www.vmware.com/products/cloud-infrastructure/vsphere-foundation)

![image](https://github.com/user-attachments/assets/ed61a986-ea2c-4d79-9d87-7e39cc2cca72)